app/konnect/gateway-manager/plugins/update-custom-plugin.md

```diff
@@ -0,0 +1,275 @@
+
+When a schema file is updated in {{site.konnect_short_name}}, the {{site.konnect_short_name}} 
+platform doesn't trigger payload reconciliation automatically.
+
+This means that if you **don't** make any configuration changes in the control plane, such as adding, 
+modifying, or deleting a {{site.base_gateway}} entity, the data plane nodes won't receive a
+payload update, and won't use the updated schema.
+
+When pushing changes to the control plane, the payload reconciliation only affects 
+data plane nodes if an instance of the plugin that uses the updated schema has its 
+configuration changed.
+
+Since plugin configurations are stored as JSON blobs, a schema change alone doesn't impact the 
+plugin configuration. However, if an instance of this plugin is also updated, the new schema affects how 
+any new plugin configuration is represented.
+
+In summary:
+* Uploading a custom plugin schema adds a new configurable object to the {{site.konnect_short_name}} plugin hub, 
+both as a tile in the UI, and an API endpoint.
+
+  You can change that schema, but changing the schema alone doesn't trigger any payloads to the data plane nodes.
+
+* The new tile or endpoint added by the schema lets you create plugin configurations.
+If you create a plugin configuration in this way, it triggers a payload reconciliation with the data plane nodes.
+
+### Custom plugin update path
+
+When you need to make plugin changes, we recommend updating the schema in 
+{{site.konnect_short_name}} first, and then on the data plane nodes:
+
+1. Start a migration/maintenance window.
+1. Update the plugin schema in {{site.konnect_short_name}}.  
+1. (Optional) Update the configuration for existing plugin instances.
+
+    Based on the nature of the schema updates, you might need this optional step 
+    after updating the schema in {{site.konnect_short_name}} to make sure that any
+    existing plugin entities are updated before the schemas are 
+    changed in the data plane nodes.
+
+1. Update the plugin schema on each data plane node.
+1. Stop the migration/maintenance window.
+
+{:.important}
+> **Important**: In cases where a breaking change is made to the schema, we **do not** 
+recommend updating data plane nodes first. 
+The change will affect each data plane node as soon as the node receives its first payload. 
+This will happen even if there are no changes in the control plane for existing or new plugins 
+using the updated schema. 
+
+See the following table for a comparison of possible changes and upgrade paths, in the case of 
+a configuration parameter change in a custom plugin's schema.
+
+**Legend:**
+
+Based on the steps defined above and your specific use case, you have to take one of 
+the following paths:
+* Short: Follow steps 1 → 2 → 4 → 5, skipping step 3.
+* Long: Follow steps 1 → 2 → 3 → 4 → 5
+* CP/DP sync required: 1 → 2 → Optionally 3, if the updated parameter is in use → 4 → 5
```

Comment:
```suggestion
* Short path: Follow steps 1 → 2 → 4 → 5, skipping step 3.
* Long path: Follow steps 1 → 2 → 3 → 4 → 5
* CP/DP sync required: 1 → 2 → Optionally 3, if the updated parameter is in use → 4 → 5
```

Ignore this if it doesn't feel right. Otherwise lgtm